macro/macroutils.py

```diff
@@ -14,6 +14,8 @@
 distro_names = ['boxturtle', 'cturtle', 'diamondback', 'electric', 'fuerte', 'groovy', 'hydro', 'indigo', 'jade', 'kinetic', 'lunar', 'melodic', 'unstable']
 distro_names_indexed = ['diamondback', 'electric', 'fuerte', 'groovy', 'hydro', 'indigo', 'jade', 'kinetic', 'lunar', 'melodic', 'unstable'] #boxturtle and cturtle not indexed
 distro_names_buildfarm = ['indigo', 'jade', 'kinetic', 'lunar', 'melodic']
+distro_names_never_on_buildfarm = ['boxturtle', 'cturtle', 'diamondback', 'unstable']
```

Comment:
I just took a guess at why these distros are being excluded from the macros, open to suggestions for a more appropriate variable name if there is one